File: 6-kyu/Multiples of 3 and 5/Multiples3n5.py
# Kata found at: https://www.codewars.com/kata/514b92a657cdc65150000006

def solution(number):
    count = 0
    for i in range(1, number):
        if i % 3 == 0 or i % 5 == 0:
            count = count + i
    return count
  

File: 6-kyu/Multiples of 3 and 5/Multiples3n5.cs
// Kata found at: https://www.codewars.com/kata/514b92a657cdc65150000006

public static class Kata
{
  public static int Solution(int value)
  {
    int count = 0;
    for(int i = 3; i < value; i++)  {
    
      if(i % 3 == 0 || i % 5 == 0)  {
      
        count += i;
      
      }
    
    }
    return count;
  }
}
